GrandCentralBoard/Widgets/Harvest/View/CircleChartViewModel.swift

```diff
@@ -18,3 +18,49 @@ struct CircleChartViewModel {
     let startAngle: Double
     let items: [CircleChartItem]
 }
+
+extension CircleChartViewModel {
+
+    static func chartItemFromBillingStats(stats: DailyBillingStats) -> CircleChartViewModel {
```

Comment:
I did that in a next PR I will open right after that (had to rewrite this class anyway).